packages/fast-usdc-contract/src/exos/advancer.ts

```diff
@@ -68,11 +72,18 @@ interface AdvancerVowCtx {
   txHash: EvmHash;
 }
 
-const AdvancerVowCtxShape: TypedPattern<AdvancerVowCtx> = M.splitRecord(
+/**
+ * @deprecated from before the adoption of CAIP-10
+ */
+type OldAdvancerVowCtx = Omit<AdvancerVowCtx, 'destination'> & {
+  destination: AccountId | CosmosChainAddress;
+};
+
+const AdvancerVowCtxShape: TypedPattern<OldAdvancerVowCtx> = M.splitRecord(
   {
     fullAmount: AnyNatAmountShape,
     advanceAmount: AnyNatAmountShape,
-    destination: M.string(), // AccountId
+    destination: M.or(AccountIdArgShape), // AccountId is preferred
```

Comment:
`M.or(...)` with 1 arg? typo?

does this suggest there's a missing test?

Reply:
it's a typo but I don't think it affects execution. `M.or()` doesn't assume more than one,
```
{(...subPatts: Pattern[]) => Matcher} or
```